src/tls13.c

```diff
@@ -3938,6 +3938,7 @@ static int FindPsk(WOLFSSL* ssl, PreSharedKey* psk, byte* suite, int* err)
         #endif
         }
         if ((ret == 0) && found) {
+            ssl->options.isPSK = 1;
```

Comment:
I am not sure you need to add a new option bit for this. What about using `ssl->options.noPskDheKe`?

Reply:
I wanted to be sure that we have actually negotiated PSK use. `ssl->options.noPskDheKe` doesn't look like it means we use PSK. I'm not seeing any other ways to do this. An alternative would be to check that `ssl->arrays->psk_keySz != 0` but I wanted an option that is set when the entire PSK setup is successful (I'm force pushing a change to push the bit setting to the end).